src/docs/development/tools/devtools/memory.md

```diff
@@ -38,77 +47,100 @@ in granularity:
   requiring more information.
 {{site.alert.end}}
 
-## Memory anatomy
+## Memory Anatomy
 
-Use a timeseries graph to visualize the state of the Flutter memory
+A timeseries graph is used to visualize the state of the Flutter memory
 at successive intervals of time. Each data point on the chart
 corresponds to the timestamp (x-axis) of measured quantities (y-axis)
 of the heap, for example, usage, capacity, external, garbage
 collection, and resident set size.
 
-![Screenshot of a memory anatomy page]({% asset tools/devtools/memory_anatomy.png @path %}){:width="100%"}
+![Screenshot of a memory anatomy page]({% asset tools/devtools/memory_chart_anatomy.png @path %}){:width="100%"}
+
+The major sections of the memory profiler are:
 
 <dl markdown="1">
-<dt markdown="1">**Legend**</dt>
-<dd>All collected measurements regarding the memory.
-    Clicking a legend name hides or shows that data.</dd>
-<dt markdown="1">**Range selector**</dt>
+<dt markdown="1">**Event Timeline**</dt>
+<dd>Displays DevTools events (such as Snapshot (manual and auto), Dart VM GC, user requested GC, Monitor and Accumulator Reset (button clicks) when they occurred, in relation to the memory chart timeline.</dd>
```

Comment:
The event timeline displays Dart VM and DevTools events on a shared timeline. These events can be snapshots (manual and auto), Dart VM GCs, user requested GCs, or monitor and accumulator reset actions.

Reply:
Done.